### ISSUE TITLE
Parallelize build command

### ISSUE THREAD

**Issue by HeDo88TH** [DroneDB/DroneDB#256]
Laz files are built in sequence and not in parallel.
Today we have machines with many cores, maybe we can run multiple builds in parallel to speed up the process.

**Comment by pierotofy** [comment]
True, however, it might be better to leave users in charge of launching multiple build commands (specifying a build path) instead of providing a threading mechanism, unless there's a good reason for not doing so.

**Comment by HeDo88TH** [comment]
> True, however, it might be better to leave users in charge of launching multiple build commands (specifying a build path) instead of providing a threading mechanism, unless there's some other good reason.

I'm referring to the `ddb build` command without arguments. If the user wants to build all the files he will benefit from parallel execution.
